katello/katello-backup

```diff
@@ -54,6 +54,17 @@ def run_cmd(command, exit_codes=[0])
   end
 end
 
+def create_directories(directory)
+  @dir << "/" unless @dir.end_with? "/"
+  @dir << "katello-backup-" + DateTime.now.strftime('%Y%m%d%H%M%S')
+  puts "Starting backup: #{Time.now}"
```

Comment:
I think the "Starting backup" message should live outside of `create_directories`, as the fact that the backup is starting has nothing to do with the creation of the folder.